ports/geckolib/glue.rs

```diff
@@ -356,7 +356,7 @@ pub extern "C" fn Servo_StyleSet_Drop(data: *mut RawServoStyleSet) -> () {
 }
 
 pub struct GeckoDeclarationBlock {
-    pub declarations: Option<PropertyDeclarationBlock>,
+    pub declarations: Option<Arc<PropertyDeclarationBlock>>,
```

Comment:
Yeah, I mean refcounting via FFI. But it isn't quite realistic either, given there are some additional fields in this struct.

I wonder whether it is feasible to make all those things only hold a immutable reference to `PropertyDeclarationBlock`, so that we don't need to wrap it with `Arc`? If not, probably we need to live with the double indirection, then.